.github/workflows/woke.yml

```diff
@@ -0,0 +1,22 @@
+name: 'woke'
+on:
+  - pull_request
+jobs:
+  woke:
+    name: 'woke'
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout'
+        uses: actions/checkout@v2
+
+      - uses: jitterbit/get-changed-files@v1
+        id: files
+
+      - name: 'woke'
+        uses: get-woke/woke-action@v0
+        with:
+          # Cause the check to fail on any broke rules
+          fail-on-error: true
+          # See https://github.com/marketplace/actions/get-all-changed-files
+          # for more options
+          woke-args: ${{ steps.files.outputs.added_modified }}
```

Comment:
wonder if this can be a tekton task instead, wdyt @prestonvasquez?

we are trying to dogfood tekton in our CI as much as possible

Reply:
@jerop That makes sense to me. Is there documentation for adding a tekton task to the pipeline repository? Would it just require extending the test/presubmit-tests.sh to include a `check_woke` function ?

Reply:
I recommend sharing the task in the tekton catalog - https://github.com/tektoncd/catalog

Then you can set up the test in plumbing - https://github.com/tektoncd/plumbing/tree/main/tekton/ci

@chitrangpatel @Yongxuanzhang could you please help @prestonvasquez through this? 🙏🏾

Reply:
@prestonvasquez it seems like you have a job here which is referencing Actions. Repeating it successfully in Tekton is not trivial but well worth the effort if we can get it to run successfully. 

My suggestion is to split this into three Tasks:
1. git-clone + checkout (no need here since we already have lots of those).
2. `get-changed-files`: you might want to construct a Task here that runs this action. (I can help with that offline).
3. `woke-action`: again you might want to construct a Task that runs this action. 

My guess is that you will have to try and mimic the environment/file system somewhat in order to get it working successfully.

Reply:
Once we have these Tasks working, we can write a pipeline to stitch them together. Once we have a pipeline, then we can get it running in our dogfooding cluster.

Reply:
I do think we could do both in parallel 👼🏼 We shouldn't refrained ourselves to use GitHub Workflows today as it helps use solve quickly something. A good example for this is the `chains` [e2e workflow](https://github.com/tektoncd/chains/actions/runs/6509483413). 
We could have the checks today, and as long as we track the work on porting this to tekton, I think it's fine to merge.

Reply:
Yeah we can merge this and try to replace with a task to move faster

Reply:
Sounds good, the commits are squashed. I can try to follow up on the following two issues. Is there a protocol for this?

Reply:
I think next step would be to create some issue in plumbing and start experimenting with it.

Reply:
Sounds good to me, thanks all